pyperf/_utils.py

```diff
@@ -193,9 +193,7 @@ def python_has_jit():
     implementation_name = python_implementation()
     if implementation_name == 'pypy':
         return sys.pypy_translation_info["translation.jit"]
-    elif implementation_name in ['graalpython', 'graalpy']:
-        return True
-    elif hasattr(sys, "pyston_version_info") or "pyston_lite" in sys.modules:
+    elif implementation_name in {'graalpython', 'graalpy'} or (hasattr(sys, "pyston_version_info") or "pyston_lite" in sys.modules):
```

Comment:
I dislike this line. It just makes the line longer and make complicated to read. I like to fit into 80 columns.